### PR TITLE
E2E screenshot tests

### PR DESCRIPTION
**Pull request by krystalleon**
# Issue/Ticket

CLOSES #XX

If you specify CLOSES, the ticket will be automatically closed and moved to 'done' when the PR is merged.

If you specify PROGRESS ON, a reference will be created to that issue (in both directions), but merging the PR will not close the issue.

# Description

Screenshot E2E testing with puppeteer for auto-app-sync feature

## Type of change

Chore

Label your PR accordingly.

# How Has This Been Tested?

# Notes

### CODE REVIEW

**Comment by alfredovargasjr** [comment]
Do we want to add these test results to the repo?

File: E2E/Auto App Sync/ScreenshotTest.js [hunk 1]
```diff
@@ -0,0 +1,376 @@
+const puppeteer = require ('puppeteer');
```
Comment (alfredovargasjr) [inline E2E/Auto App Sync/ScreenshotTest.js:1]: You can leave this in the PR but I dont think we need the png files of the results.

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]

File: Backend/UnitTesting/SessionServiceUT.cs [hunk 2]
```diff
@@ -105,31 +105,31 @@ public void Create_Session_Fail_ExceptionThrown()
...
-                _db.SaveChanges();
-                var result = _db.Sessions.Find(expectedResponse.Id);
-
-                // Assert
-                Assert.IsNotNull(response);
-                Assert.IsNull(result);
-                Assert.AreEqual(response.Id, expectedResponse.Id);
-            }
-        }
+        //[TestMethod]
```
Comment (alfredovargasjr) [inline Backend/UnitTesting/SessionServiceUT.cs]: Oh these test updates still need to be added to the master branch, :/

Comment (alfredovargasjr) [reply Backend/UnitTesting/SessionServiceUT.cs]: For now, remove these changes from your PR, sorry

**Review by alfredovargasjr — DISMISSED** [review]

**Review by julianpoy — APPROVED** [review]
Looks good. We can develop this framework further, but thanks for posting a scaffold.

**Pull request merged**